cmd/autoheal/alerts_worker.go

```diff
@@ -88,9 +88,17 @@ func (h *Healer) startHealing(alert *alertmanager.Alert) error {
 	activated := make([]*monitoring.HealingRule, 0)
 	h.rulesCache.Range(func(_, value interface{}) bool {
 		rule := value.(*monitoring.HealingRule)
-		if h.checkConditions(rule, alert) {
+		ok, err := h.checkRule(rule, alert)
+		if err != nil {
+			glog.Errorf(
+				"Error while checking if healing rule '%s' matches action '%s': %s",
```

Comment:
Should this say `matches alert` instead of `matches action`?

Reply:
Yes, I will change it.